Formula/ib.rb

```diff
@@ -0,0 +1,17 @@
+class Ib < Formula
+  desc "C/C++ build tool that automatically resolves included source"
+  homepage "https://github.com/JasonL9000/ib"
+  url "https://github.com/hooddanielc/ib/archive/v0.7.tar.gz"
+  sha256 "2bb8c6b456fb39d0b63fc40d96e4f494c3599ebc39ae1609601b9be1b2ef036f"
+
+  depends_on :python
```

Comment:
It appears to be Python 2 -only for now (https://github.com/JasonL9000/ib/pull/4).